apm-sniffer/apm-sdk-plugin/rocketMQ-5.x-plugin/pom.xml

```diff
@@ -39,6 +39,12 @@
 			<artifactId>rocketmq-client</artifactId>
 			<version>${rocketmq-client.version}</version>
 			<scope>provided</scope>
+			<exclusions>
+				<exclusion>
+					<groupId>io.grpc</groupId>
+					<artifactId>grpc-netty</artifactId>
+				</exclusion>
+			</exclusions>
```

Comment:
I think we should exclude grpc from agent-core, rather than from this side?

Reply:
Why? GRPC is needed in agent side, how can we exclude it there?

Reply:
I think the agent plugin depends on the agent core lib, which has the gRPC dependency overriding the version.
We just need the RocketMQ plugin compiled. So, about exclude, I mean for this plugin, it could depend on agent-core with gRPC excluded. In the runtime, this plugin is using the user's gRPC, our gRPC and netty would have been shaded in another package already. So, we should be good by that.